### ISSUE TITLE
ValueError: too many values to unpack (expected 2)

### ISSUE THREAD

**Issue by jayvdb** [moremoban/moban#297]
https://travis-ci.org/jayvdb/coala-bears/jobs/560303678
```
$ moban
Warning: 'copy:' is deprecated since 0.4.0! Please use 'targets:' instead
Cloning https://gitlab.com/coala/package_manager/...
Cloning https://gitlab.com/jayvdb/mobans...
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/bin/moban", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/moban/main.py", line 34, in main
    count = handle_moban_file(moban_file, options)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/moban/main.py", line 149, in handle_moban_file
    mobanfile.handle_moban_file_v1(moban_file_configurations, options)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/moban/mobanfile/__init__.py", line 66, in handle_moban_file_v1
    expand_template_directories(merged_options[constants.LABEL_TMPL_DIRS])
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/moban/plugins/template.py", line 177, in expand_template_directories
    yield expand_template_directory(directory)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/moban/plugins/template.py", line 184, in expand_template_directory
    library_or_repo_name, relative_path = directory.split(":")
ValueError: too many values to unpack (expected 2)
```

yaml is

```yaml
requires:
  - https://gitlab.com/coala/package_manager/
  - type: git
    url: https://gitlab.com/jayvdb/mobans
    branch: bears-pr-final

configuration:
  template_dir:
    - .moban.dt/
    - 'mobans:templates/'
    - 'mobans:mobans:assets'
    - 'package_manager:'
  configuration: .moban.yaml
  configuration_dir: 'mobans:'
targets:
```

problem appears to be `'mobans:mobans:assets'`

**Comment by chfw** [comment]
- 'mobans:mobans:assets'？

**Comment by chfw** [comment]
OK. Shall we defend it then?

**Comment by jayvdb** [comment]
Ya, users should never see an exception.

**Comment by jayvdb** [comment]
Replacing `'mobans:mobans:assets'` with `'mobans:assets/'` fixed it.

**Comment by chfw** [comment]
@jayvdb , I am sorry that I have seen your tag. My train took me offline.

**Comment by chfw** [comment]
I agree with you that we shall leave some issues for new joiners. I advocated the same thing in other github projects where I was involved.

**Comment by chfw** [comment]
## initial solutions

### git repo syntax
```
    requires:
       - https://github.com/moremoban/pypi-mobans
    configuration:
      template_dir:
        - "repo://pypi-mobans/templates"
        - local
      configuration: config.yml
      configuration_dir: "repo://pypi-mobans/config"
    targets:
      - mytravis.yml: travis.yml.jj2
      - test.txt: demo.txt.jj2
```

### pypi syntax

```
    requires:
       - pypi-mobans-pkg
    configuration:
      template_dir:
        - "pypi://pypi-mobans-pkg/templates"
      configuration: config.yml
      configuration_dir: "pypi://pypi-mobans-pkg/config"
    targets: 
      - mytravis.yml: travis.yml.jj2
      - test.txt: demo.txt.jj2
```

**Comment by chfw** [comment]
will be fixed in 0.6.0
